program-workflows/hiv-recency-testing.md

```diff
@@ -64,9 +64,98 @@ The reporting tools include the national HMIS tools are:
 * HTS Client Card Export - CSV file containing all the fields entered on the HTS Card 
 * Recency Data Export - CSV file with recency data fields 
 
+
+
 ## Data Sharing with Central Server
 
-This is an automated process that occurs every hour until a successful submission is done in a day. Due to the small numbers involved, all the HTS records are sent each time, a process which will be improved as the program proceeds
+This is an automated process that occurs every hour until a successful submission is done in a day. Due to the small numbers involved, all the HTS records are sent each time, a process which will be improved as the program proceeds. The EMR gives the option to change the scheduled sync frequency.
+
+#### Sync data flow
+
+![](/assets/sync-data-flow.png)
+
+A few key points to note:
+
+* **Facility surveillance** data is collected in an EMR. Through an internet connection, the data will be sent to the Central Database at CPHL on regular intervals \(every day\).
+
+* **UVRI data** is collected by the UVRI data management system. UVRI to send the data as excel files once a week to the Central database at CPHL. A web or file transfer interface will be used to upload the excel file with the previous week’s data.
+
+* **CQI data** for all sites will be collected using ODK. This data will then be uploaded into the Power BI Analysis Database.
+
+* **The central database** will hold all the data and combine/integrate the two datasets \(EMR and UVRI data\) into a single Recency dataset.
+
+* **The Power BI Analysis Database** will disclose this data through dashboards to authorized users
+
+
+
+#### Acquiring credentials
+
+During or prior to recency activation at each site, METS/UCSF provide each facility with authentication details that can be added within the facility’s EMR. The credentials are created within MIRTH. MIRTH is the system that receives data from all facilities and verifies the credentials. It is installed at CPHL central server.
+
+The username is preferred as the national DHIS2 ID. The password is a random set of characters and numbers.
+
+#### Sync authentication
+
+The supported authentication methods are stated in the API description below.
+
+#### API description
+
+The description of the API that receives the data can be found at this link.
+https://github.com/METS-Programme/hie-documentation
+
+
+
+#### Data dictionary
```

Comment:
@mugumedavid Remove this and add a link to this https://github.com/METS-Programme/hie-documentation/blob/master/reference/recency-surveillance.v1.json

Reply:
Perhaps then the API documentation should describe the column attributes in more details. For example, `health_unit_sub_county` actually takes the client address sub-county and default to facility sub-county if address information is not available.

Reply:
@mugumedavid that is done during transformation not data collection - I also think that is an incorrect assumption leading to misleading results for hotspot placement